src/shipit_code_coverage/tests/test_uploader.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import requests
+import responses
+
+from shipit_code_coverage import uploader
+
+
+@responses.activate
+def test_get_codecov(mock_secrets, codecov_commits):
+    data = uploader.get_codecov('0548f006a32c138218ad586177db83486c82a09e')
+    assert data['commit']['state'] == 'complete'
+
+    with pytest.raises(requests.exceptions.HTTPError):
+        uploader.get_codecov('7d7b21f5ede24dbdd59e17980a332f4b240f8e13')
```

Comment:
I'm not sure I understand this `with` statement properly: Does it assert that an `HTTPError` is actually thrown? Shouldn't you check that the return value of `uploader.get_codecov` is `None`?

Reply:
Yes, it asserts that `HTTPError` is thrown (it's thrown by https://github.com/mozilla-releng/services/pull/1122/files#diff-0db15d7e275ece3900d2c3f084332f1fR75, as when a commit is missing codecov.io returns 404).